src/models/power-roll.ts

```diff
@@ -0,0 +1,16 @@
+import type { Characteristic } from '../enums/characteristic';
+import type { PowerRollType } from '../enums/power-roll-type';
+
+interface _PowerRoll {
+	characteristic: Characteristic[];
+	bonus: number;
+	tier1: string;
+	tier2: string;
+	tier3: string;
+}
+
+export interface Test extends _PowerRoll {
+	type: PowerRollType.Test
+}
+
+export type PowerRoll = (_PowerRoll & { type: PowerRollType.PowerRoll}) | Test;
```

Comment:
Should the "standard" power roll be renamed to `AbilityPowerRoll`, to distinguish it from `Test`?

I want to keep the separate `Test` export to restrict its use: so a "standard" PowerRoll can't be used as a Malice test.

It seems like we'd also want the opposite, too: only allow AbilityPowerRoll to be placed on an ability, and not a Test.

Reply:
that is:

```typescript
export interface AbilityPowerRoll extends _PowerRoll {
  type: PowerRollType.AbilityPowerRoll;
}

export type PowerRoll = AbilityPowerRoll | Test;
```

and then most existing things that can have a `PowerRoll` will instead only have a `AbilityPowerRoll` specifically, disallowing a `Test` there

Reply:
This all might be simpler if we drop the concept of a power roll "type" altogether, and just have the `header` value passed in to the `PowerRollPanel` determined by the calling context?

(So the Malice including a `PowerRollPanel` for its `test?: PowerRoll` will create the "Intuition Test" header, for example.)